src/test/java/rolling/AdvancementCalcTest.java

```diff
@@ -0,0 +1,28 @@
+package rolling;
+
+import advancement.GeneralSkillCalculator;
+import advancement.SpecialtySkill;
+import com.google.common.collect.ImmutableList;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+class AdvancementCalcTest {
+
+    @Test
+    void testTall() {
+        final List<SpecialtySkill> specialtySkills = new GeneralSkillCalculator(ImmutableList.of(8, 4), true, true, 12, true, 8).generate(10L);
+        final int sum = specialtySkills.stream().flatMapToInt(specialtySkill -> specialtySkill.getAPLog().stream().mapToInt(value -> value)).sum();
+        Assertions.assertEquals(108, sum);
+        System.out.println(specialtySkills);
```

Comment:
Why are you printing this out? Rather than just doing a proper assert against the list?

Reply:
That was for me doing stuff in debug mode before I converted it into a command. Must have forgotten to remove it.